### PR TITLE
feat: Add Prefix to Transaction Export

### PR DESCRIPTION
**Pull request by dhaselhan**
* Fix and update transaction tests
* Fix user role check to use correct values intsead of the name of the enum

### CODE REVIEW

File: backend/lcfs/db/models/transaction/Transaction.py [hunk 1]
```diff
@@ -6,6 +6,18 @@
 
 from lcfs.db.base import BaseModel, Auditable, EffectiveDates
 
+transaction_type_to_id_prefix_map = {
```
Comment (AlexZorkin) [inline backend/lcfs/db/models/transaction/Transaction.py:9]: I would like to move these into the /utils/constants.py file. Transactions are not directly related to entities on the backend so better to keep these references separate. This is one of those annoying moments where Transactions on the backend don't necessarily mean the same thing as on the frontend.

**Review by AlexZorkin — CHANGES_REQUESTED** [review]
Looks great, let me know what you think of moving the mappings.

File: backend/lcfs/db/models/transaction/Transaction.py (hunk 1 above)
Comment (dhaselhan) [reply backend/lcfs/db/models/transaction/Transaction.py]: Done! Moved to constants.py. Wasn't sure if I should make it a class / uppercase it so let me know.

**Review by AlexZorkin — APPROVED** [review]
Mappings look great ty.

**Pull request merged**